pkg/virtctl/vnc/vnc.go

```diff
@@ -59,6 +60,8 @@ const (
 	TIGER_VNC     = "vncviewer"
 )
 
+var proxyOnly bool
```

Comment:
We should not have started creating gloabls for these flags. They could just have been checked in the actual command. Can be changed for all commands in a follow-up. No need to change it here.